### PR TITLE
DB Consistency Updates for AB#16448

### PR DESCRIPTION
**Pull request by sslaws**
# Fixes or Implements [AB#16448](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/16448)

## Description

Updates the dashboard queries to be consistent with naming and use DateOnly instead of strings
Removed custom mapping and HG Function to Postgres date_trunc function 
Updated Init script to include the latest migrations
Removed hard-coded query parts in Refit API
Minor analyser fixes
Fixes to Source names for Telemetry

## Testing

- [X] Unit Tests Updated
- [ ] Functional Tests Updated
- [ ] Not Required

## UI Changes
None

## Notes

I did remove the initialLoad boolean from RetrieveData in the client as it didn't look required and it still behaves properly.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Apps/Admin/Server/Controllers/DashboardController.cs [hunk 1]
```diff
@@ -117,16 +117,16 @@ public IActionResult GetDependentCount(int timeOffset)
...
-            return new JsonResult(this.dashboardService.GetRecurrentUserCounts(days, startPeriod, endPeriod, timeOffset));
+            return new JsonResult(this.dashboardService.GetRecurrentUserCounts(days, startDateLocal, endDateLocal, timeOffset));
         }
 
         /// <summary>
         /// Retrieves the ratings summary.
         /// </summary>
-        /// <param name="startPeriod">The period start to calculate the summary.</param>
-        /// <param name="endPeriod">The period end to calculate the summary.</param>
+        /// <param name="startDateLocal">The starting date to get the user counts in the client's Local time.</param>
```
Comment (MikeLyttle) [inline Apps/Admin/Server/Controllers/DashboardController.cs:128]: This action doesn't return user counts.

File: Apps/Admin/Client/Pages/AnalyticsPage.razor.cs
Comment (MikeLyttle) [inline Apps/Admin/Client/Pages/AnalyticsPage.razor.cs:1]: The TimeOffset property in this file should no longer be multiplied by -1.

Comment (MikeLyttle) [reply Apps/Admin/Client/Pages/AnalyticsPage.razor.cs:1]: Not addressed in latest update.

Comment (sslaws) [reply Apps/Admin/Client/Pages/AnalyticsPage.razor.cs:1]: Good catch!

**Review by MikeLyttle — APPROVED** [review]

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**